frontend/src/components/Mobile/InputField.tsx

```diff
@@ -1,24 +1,45 @@
 import React, { Dispatch, SetStateAction, useState } from 'react'
 import styled from 'styled-components'
+import { FieldValues, UseFormRegister } from 'react-hook-form'
 
 import { Input } from 'antd'
 
+type InputFieldProps = {
```

Comment:
move this to after styled components instead

Reply:
[d3f894c](https://github.com/rhdevs/rhapp/pull/614/commits/d3f894cadaa0993923546ac8e1102cfa56d1ffac)